go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/aquasecurity/harbor-scanner-trivy
 go 1.13
 
 require (
+	github.com/aquasecurity/trivy v0.5.3-0.20200317001709-3c9af62fb880
```

Comment:
Note to self: Update this to master once https://github.com/aquasecurity/trivy/pull/435 is merged.

Reply:
I think we should bump up trivy in Dockerfile as we currently rely on its CLI interface

Reply:
Got it. I'll wait until this gets approved by @knqyf263 then.